.gitignore

```diff
@@ -1,3 +1,3 @@
-vendor
-composer.lock
```

Comment:
I think it is better to add the lockfile now, to have wider spectrum of tests, so we can make sure the library is compatible with all dependencies. Please have a look on comments here:
https://github.com/zendframework/zend-inputfilter/pull/107#issuecomment-221928492